specification/src/chapter2.adoc

```diff
@@ -265,12 +270,17 @@ the same way to confidential workloads.
 | Lower privileged software MUST NOT be able to deny service to higher
 privileged software, or other isolated workloads at the same privilege level.
 
+| SR_DOS_002
+| Software in one domain SHOULD NOT be able to deny service to software in a different domain
```

Comment:
this may not always be true - in a CC use case, a hosting domain which owns all resources (from a mgmt perspective), can and should be able to deny service to software in a seperate domain that hosts confidential workloads.

Reply:
There is (should be) a comment to that effect. And this is why it is SHOULD.

I'll check. And I'll look at if we can replace these too with references instead.